### PR TITLE
Fix for hotkeys still being applied when cleared (my fault `not` wher…

### PR DESCRIPTION
**Pull request by sebastiansam55**
…e there shouldn't have been one)

### CODE REVIEW

**Comment by BlueDrink9** [comment]
@josephj11 good to merge

**Comment by BlueDrink9** [comment]
Probably on my head for not cloning and checking more thoroughly, but I was expecting @sebastiansam55 to have run tests before pushing. This commit breaks 4 tests. I'm looking into why now.

I've added a note to the CONTRIBUTERS.rst file on my `develop` branch suggesting creating a `pre-push` git hook to run them all. On my machine it takes about 10 seconds to run the full suite, so it's worth doing.

**Comment by josephj11** [comment]
I could revert it if desired, but you'd have to tell me how to do that.

**Comment by BlueDrink9** [comment]
I'm sorting out the underlying reason why these two PRs are causing issues. This isn't the only one; #451 is also causing issues. I'm not sure we need to just yet, since it's develop and the tests are related to new features no-one seems to be using. I'll fix it in another PR.

I see github is offering a 'revert' option beside the merge commit on this PR, if we need it.

File: lib/autokey/model/abstract_hotkey.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def set_hotkey(self, modifiers, key):
         modifiers.sort()
         self.modifiers = modifiers
         self.hotKey = key
-        if key is not None and TriggerMode.HOTKEY not in self.modes:
+        if key is not None and TriggerMode.HOTKEY in self.modes:
```
Comment (BlueDrink9) [inline lib/autokey/model/abstract_hotkey.py:48]: What is the intended behavior here? Currently your 'fix' duplicates the HOTKEY triggermode by adding it only if it already exists, is that intentional?

Comment (BlueDrink9) [reply lib/autokey/model/abstract_hotkey.py:48]: @sebastiansam55

**Pull request merged**